package.json

```diff
@@ -24,7 +24,9 @@
         "typescript": "^4.5.5"
     },
     "scripts": {
-        "build": "tsc --build packages/* test",
+        "build": "npm run build:esm && npm run build:cjs",
+        "build:esm": "tsc --build packages/* test",
+        "build:cjs": "tsc -p packages/parse5/tsconfig.cjs.json && echo '{\"type\":\"commonjs\"}' > packages/parse5/dist/cjs/package.json",
```

Comment:
does this make node think `dist/cjs/` is its own package?

Reply:
This will make `require('parse5')` work in Node versions with ESM support.

Because the files are plain `.js` files, Node will look at the closest `package.json` file to determine if the code is ESM or not. We create a `package.json` for the CJS code, to make sure requiring it works as expected.

Reply:
I can't think of any side-effects, but definitely something worth keeping in mind for future bug-reports. I adopted the pattern from [this article](https://www.sensedeep.com/blog/posts/2021/how-to-create-single-source-npm-module.html) from SenseDeep, which is used in several published (but not very popular) modules.